random.cabal

```diff
@@ -83,14 +95,17 @@ library
     hs-source-dirs:   src
     default-language: Haskell2010
     ghc-options:
-        -Wall -Wincomplete-record-updates -Wincomplete-uni-patterns
```

Comment:
Are these warning not available in the old GHC version? I think they are valuable. If they are only available in newer versions, let's activate them in newer versions.

.. or alternatively, leave them as they are here and overwrite `ghc-options` in the existing `if impl(ghc < 8.0)` block below.

Reply:
I had a feeling you are going to say this :smile: That's right, they are not available on older ghc. I am usually all for theses warnings, but we aren't using records or pattern matches that might fail. Regardless of that fact, I'll add them back with a condition.